### ISSUE TITLE
Expand arrow gone from summary element

### ISSUE THREAD

**Issue by vanillajonathan** [necolas/normalize.css#640]
The visual expand arrow visible in Chrome and Firefox on the `details` element disappears with normalize.

I don't know if this is intentional or not.

```html
<details>
  <summary>Click on this summary to expand</summary>
  <p>Here is additional text, visible when expanded...</p>
</details> 
```

http://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_details

**Comment by thierryk** [comment]
On what browser/OS? Because I don't see that issue on my end (Chrome 53/MacOS Sierra).

**Comment by vanillajonathan** [comment]
Normalize.css
https://jsfiddle.net/xa9bbr96/1/

Bootstrap 3.3.7
https://jsfiddle.net/vu6o29es/

Seems this is not reproducable wirth `normalize.css`, but under Bootstrap it is reproducable in Chrome 54 and Firefox 49 on Windows 10.

Developer says line 33 in `normalize.less` which is:

    article, aside, details, figcaption, figure, footer, header, hgroup, main, menu, nav, section, summary { display: block }

And when I uncheck that rule in the developer tools then the arrow appears.

**Comment by patrickhlauke** [comment]
Can't repro in Chrome 54, but I do see it in Firefox 49. Removing the `display:block` makes the default triangle reappear in Firefox.

**Comment by thierryk** [comment]
This is not an issue with normalize but with bootstrap. See my comments here: https://github.com/twbs/bootstrap/issues/21060

**Comment by awayken** [comment]
I agree with @thierryk.

I was seeing this issue, too. Tracked it down to Firefox giving `<summary>` using `display: list-item` instead of `display: block`, which is what I was seeing in Normalize. Then I realized that I was using Normalize 4.0. This normalization is fixed in Normalize 5.0 where it uses `display: list-item`.

Seems like this issue can be closed.
